src/components/settings/ReportsTab.vue

```diff
@@ -60,11 +112,32 @@ export default {
   data() {
     return {
       config: this.configItem,
+      reportLogo: false,
+      logoPath: "",
+      chosenFile: null,
     };
   },
+  created() {
+    if (!this.config.logo) {
+      this.config.logo = {
```

Comment:
@dockabernathy01 Rather than adding this here, can you add the logo data to the default config like mentioned [here](https://github.com/yatt-ai/yattie/pull/146#discussion_r1534244078)

Reply:
I will update this.